app/helpers/article_helper.rb

```diff
@@ -15,10 +15,14 @@ def link_subjects(options = {})
     return unless options[:value]
     separators = options.dig(:config, :separator_options) || {}
     values = render_text_from_html(options[:value])
-    values.collect do |value|
+    if params[:action] == 'index'
```

Comment:
Hmm, this feels a bit wrong.  Perhaps we can discuss alternate solutions at some point today.